src/sentry/contexts.cpp

```diff
@@ -73,6 +73,7 @@ Dictionary make_device_context(const Ref<RuntimeConfig> &p_runtime_config) {
 		device_context["model"] = model;
 	}
 
+	// Note: Godot Engine doesn't support changing screen resolution.
```

Comment:
Like, ever? How do Godot games support different screens?

Reply:
Godot doesn't change the physical resolution of your desktop environment, it changes resolution scale instead. I.e., projects 720p on 1080p resolution, for example, like consoles do. Most modern games do the same, some also support changing the real resolution. But this way it makes `Alt-Tab` faster, and fewer issues with it. Upscaling is cheap these days.

Reply:
Yep. The rendering resolution is referred to as the viewport size (of the window), and you can have multiple windows as well, on different displays - it can get really complicated. You can also have nested viewports, so technically, the game could be rendered at higher resolution nested viewport, downscaled to window viewport size and upscaled to main screen resolution. :smile:  Not that I recommend it.
Reference to doc: https://docs.godotengine.org/en/stable/tutorials/rendering/multiple_resolutions.html